isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/konzept/inhalt.adoc

```diff
@@ -367,28 +371,30 @@ Die Authentifizierung wird von der eigentlichen fachlichen Anfrage getrennt, so
+Die folgende Abbildung zeigt die zukünftig intendierte System-Architektur, die bei Aufrufen von externen Anwendungen zum Einsatz kommen soll.
 
 .Authentifizierung am Authentication-Gateway
 [id="image-servicegateway-plus-authgateway",reftext="{figure-caption} {counter:figures}"]
 image::isy-security:konzept/ServiceGateway_plus_AuthGateway.dn.svg[align="center"]
 
-Damit ein Client einer externen Anwendung auf die Ressourcen einer Anwendung zugreifen kann, benötigt er ein gültiges Access-Token.
-Dieses Token muss sich der Client zuerst beschaffen, indem er sich beim _IdentityAccessManagement-Service (IAM-Service)_ authentifiziert.
-Mit der Einführung des _Authentication-Gateways_ werden somit alle Anfragen mit Authentifizierungsdaten über einen separaten Kommunikationskanal geleitet.
-Damit kommt keine mit der Authentifizierung involvierte Komponente mit späteren Anwendungsanfragen in Kontakt und umgekehrt.
+In der Abbildung sind die folgenden Ablaufschritte dargestellt:
+
+. Für die Authentifizierung einer externen Anwendung existiert ein eigener Servicepunkt, das _Authentication-Gateway_. 
+An diesen Servicepunkt werden von der extenen Anwendung die Authentifizierungsdaten übergeben.
+. Das _Authentication-Gateway_ leitet die Authentifizierungsdaten an den _IAM-Service_ weiter.
+. Der _IAM-Service_ prüft die Authentifizierungsdaten und liefert dem  _Authentication-Gateway_ ein Access-Token zurück.
+. Das _Authentication-Gateway_ übergibt das Acess-Token der externen Anwendung.
+. Die externe Anwendung kann dieses Access-Token speichern und es anschließend für _mehrere_ fachliche Anfragen an das Anwendungssystem (über das _Service-Gateway_) verwenden.
+. Für jede dieser fachlichen Anfragen wird das jeweils mitgelieferte Access-Token validiert. Das aufgerufene Anwendungssystem prüft hierzu dessen Signatur, dessen Gültigkeit und die im Access-Token hinterlegten Berechtigungen,
+  die im Hinblick auf den jeweils aufgerufenen Service vollständig vorhanden sein müssen.
 
-Die externe Anwendung sendet ihre Authentifizierungsdaten an das _Authentication-Gateway_, das die Anfrage an den IAM-Service weiterleitet.
-Im IAM-Service erfolgt dann mit den übergebenen Authentifizierungsdaten die eigentliche Authentifizierung.
-Dabei wird neben der neuen Authentifizierungslogik des _Client-Credential-Flows_ auch die bisherige Authentifizierungslogik des _Resource-Owner-Password-Credentials-Flows_ unterstützt.
-Der _Client-Credential-Flow_ dient dabei der Authentifizierung eines technischen Clients und über den _Resource-Owner-Password-Credentials-Flow_ erfolgt die Authentifizierung eines technischen Nutzers.
-Mit validen Authentifizierungsdaten und erfolgreicher Authentifizierung liefert der IAM-Service dann ein valides, neu erstelltes Access-Token an die externe Anwendung zurück.
+Die neue System-Architektur hat den Vorteil, dass sie das Service-Gateway von der Aufgabe befreit, 
```

Comment:
Diesen Vorteil würde ich entfernen, da der Hintegrund der Trennung ein ganz anderer ist, hier aber nicht beschrieben werden muss.

Reply:
Auf Nennung des "separation of concerns" Vorteils wurde verzichtet